### PR TITLE
Loop timer

### PR DESCRIPTION
**Pull request by jamiahuswalton**
I added a game loop timer and I want to know if this is the best way to approach it.

### CODE REVIEW

File: Program.cs [hunk 1]
```diff
@@ -153,32 +178,34 @@ static void Main(string[] args)
                 "finding the axe and cutting down the tree.\nUse w, s, a and d to move around and x to exit the game.  Enjoy!\n");
 
             // loop for continual movement
-            while (JayW.getHealth() > 0)
+            while (true)
             {
-                Console.WriteLine("Where do you want to go traveler? The X represents where you are now.\n");
-                // printing out game board
-                PrintGameBoard(tileGrid, currRow, currCol);
+                if (canRun)
```
Comment (havok13888) [inline Program.cs]: you can do if(canRun && JayW.getHealth() >0) instead

File: Program.cs [hunk 2]
```diff
@@ -1,4 +1,5 @@
 ﻿using System;
+using System.Timers; //We want to use this package to add timers to our game.
```
Comment (JLeezy93) [inline Program.cs:2]: Why did you use the '+' ?

File: Program.cs (hunk 1 above)
Comment (jamiahuswalton) [reply Program.cs]: That's a good point. I will make that change

File: Program.cs (hunk 2 above)
Comment (jamiahuswalton) [reply Program.cs:2]: I dont think there is a '+' there. I just checked it and it looks like there is nothing in front of it. Strange lol

**Review by JLeezy93 — APPROVED** [review]

**Review by JLeezy93 — APPROVED** [review]

**Review by havok13888 — APPROVED** [review]
done

**Review by havok13888 — APPROVED** [review]

**Review by JLeezy93 — APPROVED** [review]

**Review by JLeezy93 — APPROVED** [review]
ok

**Review by havok13888 — APPROVED** [review]
Take it!

**Review by JLeezy93 — APPROVED** [review]
ok

**Pull request merged**